core/services/assets/BlockAssetManager.php

```diff
@@ -212,7 +212,7 @@ public function addScript($handle, array $dependencies = array())
                 $this->domain->assetNamespace(),
                 $handle
             ),
-            $this->addDefaultBlockScriptDependencies($dependencies)
+            $dependencies
```

Comment:
Assuming that frontend context will need the same defaults as the editor context for dependencies is incorrect (especially when it comes to block editor api stuff).  We still may have a new helper method for frontend dependencies at some point (i.e. it's probably a safe bet `wp.i18n` would be one) but I'd like to iterate on that once we actually have a frontend script being built.

Reply:
should you not have to add the following dependencies for the editor blocks now?

```php
'wp-blocks',    // Provides useful functions and components for extending the editor
'wp-i18n',      // Provides localization functions
'wp-element',   // Provides React.Component
'wp-components', // Provides many prebuilt components and controls
CoreAssetManager::JS_HANDLE_EE_COMPONENTS
```

ie: removing these from the frontend context means you have also removed them from the admin context, but I don't see them being added back anywhere.

Reply:
This is just in the `addScript` method.  `addEditorScript` still has this call.